src/vm-vcpu/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "vm-vcpu"
+version = "0.1.0"
+authors = ["rust-vmm AWS maintainers <rust-vmm-maintainers@amazon.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+libc = "0.2.76"
+kvm-bindings = { version = "0.3.0", features = ["fam-wrappers"] }
+kvm-ioctls = "0.5.0"
+vm-device = { git = "https://github.com/aghecenco/vm-device.git", branch = "send" }
```

Comment:
This is prettier with the `[patch]` approach seen here: https://github.com/rust-vmm/vmm-reference/pull/32/commits/d0318fc093e71debef1d6bf8fee0911d23b57c1e

Reply:
I think we can now use the upstream one. I'll update it.

Reply:
I added a new commit which updates the vm-device dependency everywhere to use the upstream version.